pkg/kubectl/generate/versioned/secret_for_docker_registry_test.go

```diff
@@ -73,7 +73,7 @@ func TestSecretForDockerRegistryGenerate(t *testing.T) {
 			},
 			expected: &v1.Secret{
 				ObjectMeta: metav1.ObjectMeta{
-					Name: "foo-7566tc6mgc",
+					Name: "foo-548cm7fgdh",
```

Comment:
the original PR [changed this](https://github.com/kubernetes/kubernetes/pull/68441/files#diff-9afb1c73ad1f8cedbf9b93c7c1942397R76), this is reverting to the original value